packages/@react-stately/combobox/src/useComboBoxState.ts

```diff
@@ -51,6 +55,7 @@ export function useComboBoxState<T extends object>(props: ComboBoxStateProps<T>)
     shouldCloseOnBlur = true
   } = props;
 
+  let [showAllItemsTracker, setShowAllItemsTracker] = useState(false);
```

Comment:
Different name here because the `open` and `toggle` functions have `showAllItems` as a param. Maybe I could rename this to `displayAllItems`? I definitely could change this to `showAllItems` if I remove that parameter, but that's subject to the discussion in a previous comment